morango/controller.py

```diff
@@ -33,8 +33,13 @@ def serialize_into_store(self):
             for (_, klass_model) in iteritems(syncable_dict):
                 for app_model in klass_model.objects.filter(_morango_dirty_bit=True):
                     try:
-                        # set new serialized data on this store model
                         store_model = Store.objects.get(id=app_model.id)
+
+                        # if store record dirty and app record dirty, append store serialized to conflicting data
+                        if store_model.dirty_bit:
+                            store_model.conflicting_serialized_data = store_model.serialized + "\n" + store_model.conflicting_serialized_data
```

Comment:
Is this consistent with the ordering the dequeuing merge conflict appending does? I had imagined it appending the new conflicting records to the end.